packages/dcos-checks/buildinfo.json

```diff
@@ -0,0 +1,5 @@
+{
+	"state_directory": true,
+	"username": "dcos_3dt",
```

Comment:
Eek! @darkonie Shouldn't this be `dcos_checks`? Please make sure no other `3dt`s or `dddt`s slipped in.